example/gluon/embedding_learning/README.md

```diff
@@ -0,0 +1,72 @@
+# Image Embedding Learning
+
+This example implements embedding learning based on a Margin-based Loss with distance weighted sampling [(Wu et al, 2017)](http://www.philkr.net/papers/2017-10-01-iccv/2017-10-01-iccv.pdf). The model obtains a validation Recall@1 of ~64% on the [Caltech-UCSD Birds-200-2011](http://www.vision.caltech.edu/visipedia/CUB-200-2011.html) dataset.
```

Comment:
does performance match the original paper?

Reply:
Yes, approximately (slightly higher recall@1 and slightly lower recall@16 than what's reported in the paper). The difference is < 1%.

The difference between this implementation and the original implementation is that here we perform sampling within each GPU while the original paper implements cross-gpu sampling. Since the performance is almost identical (at least on this dataset), I use per-gpu sampling here for simplicity.

Reply:
@chaoyuaw Thanks for the contribution! Do you mind noting the difference with the original paper (per-gpu sampling) explicitly in readme? That will be good information to know when others reuse the example.

Reply:
Many thanks for the good suggestion! Yes, I will update this soon.